restapi/src/main/java/io/stargate/web/models/udt/UdtInfo.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright The Stargate Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.stargate.web.models.udt;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.base.MoreObjects;
+import io.swagger.annotations.ApiModelProperty;
+import java.util.List;
+import javax.validation.constraints.NotNull;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class UdtInfo {
+  private String name;
+  private boolean frozen;
+  private @NotNull List<UdtType> typeParams;
```

Comment:
I'm a little confused. UdtAdd has a list of UdtTypes called `fields` and each UdtType has an `info` of UdtInfo. Then each UdtInfo also has a list of UdtTypes called `typeParams`?

Reply:
Okay, just read through `decodeType()` and I think I understand now. It might be easier to follow if we call this `subType` or something like that.

Reply:
I have renamed to subType following the review comment by @dimas-b :)